modules/core/arrow-test/src/main/kotlin/arrow/test/laws/BimonadLaws.kt

```diff
@@ -8,27 +8,35 @@ import io.kotlintest.properties.forAll
 
 object BimonadLaws {
   fun <F> laws(BF: Bimonad<F>,
-               f1: (Int) -> Kind<F, Int>,
+               f: (Int) -> Kind<F, Kind<F, Int>>,
```

Comment:
This is still possible to make simply with Monad. It's `just(just(it))`

Reply:
Makes sense